configs/live/realsense.yaml

```diff
@@ -1,14 +1,15 @@
 Results:
-  save_results: False
+  save_results: True
   save_dir: "results"
   save_trj: False
   save_trj_kf_intv: 0
   use_gui: True
-  eval_rendering: False
+  eval_rendering: True
```

Comment:
Set `save_results` and `eval_rendering` **False** by default.

Reply:
Done